piechart/Pie.js

```diff
@@ -133,10 +133,14 @@
                     color: this._color(i)
                 }, this.options.sliceOptions);
 
-                var slice = L.circle(this._latlng, options).addTo(this);
+                this._data[i].slice = L.circle(
+                   this._latlng,
+                   this.options.radius, 
+                   L.Util.extend({}, options, this.options.pathOptions)
```

Comment:
@riltim Hmm, sorry I missed this before, but this seems to have 3-space indentation. Also: L.Circle now accepts `radius` as an option, no need to specify the radius explicitly as second argument.